New Entry: General MFA (#1007)
Codecademy/docs: content/general/concepts/mfa/mfa.md

---

> Title: 'MFA'

Author: Can do~

Description: 'Multi-factor Authentication (MFA) is the use of more than one factor to prove the identity of a user.'
Subjects:

>   - 'Web Development'
>   - 'Computer Science'

Reviewer: ```suggestion
  - 'Computer Science'
  - 'Cybersecurity'
```

Wouldn't this concept more closely related to the field of cybersecurity as opposed to web development?

Tags:
  - 'Best Practices'
  - 'Cybersecurity'
CatalogContent:

>   - 'paths/code-foundations'
>   - 'paths/computer-science'

Reviewer: ```suggestion
  - 'introduction-to-cybersecurity'
  - 'paths/full-stack-engineer-career-path'
```

Both of the these pages contain lessons and modules pertaining to user authentication/authorization.

---

**Multi-factor Authentication (MFA)** is the use of more than one factor to prove the identity of a user.

## Factors

One factor is typically a username or email and password combination. This is the basic log-in functionality that most websites require user's to create when initially signing up for an account.

> Other factors, depending on the verification system, can be a physical item, pin, key or token, biometric marker, or location.

Reviewer: Wondering if we can turn this into an unordered list?

```suggestion
Other factors, depending on the verification system, include the following:

- A physical item belonging to the user (e.g., a mobile phone).
- A password, personal identity number (PIN), key, or token that only the user should know.
- Some biometric marker for the user (fingerprints, voice-recognition, key-stroke patterns, etc.).
- Information about the user's [geolocation](https://www.codecademy.com/resources/docs/general/geolocation).
```

## Examples

> ATMs often require a physical card and a memorized pin to successfully take out cash.

Reviewer: ```suggestion
ATMs often require a physical card and a memorized PIN to successfully withdraw cash.
```

A secure sites, like GitHub, allow users to enable MFA through a pin that's sent to a physical device like a mobile phone. This is very secure because in order to be verified, a person would need the physical device which probably has a pin and biometric marker factor.

> More factors in the verification process results in greater security.

Reviewer: ```suggestion
More factors in the verification process results in greater [security](https://www.codecademy.com/resources/docs/general/cybersecurity).
```
